d2bs/kolbot/D2BotFollow.dbj

```diff
@@ -12,7 +12,7 @@ var StarterConfig = {
 	CDKeyInUseDelay: 5, // Minutes to wait before connecting again if CD-Key is in use. SwitchKeys overrides this!
 	ConnectingTimeout: 20, // Seconds to wait before cancelling the 'Connecting...' screen
 	PleaseWaitTimeout: 30, // Seconds to wait before cancelling the 'Please Wait...' screen
-	WaitInLineTimeout: 60, // Seconds to wait before cancelling the 'Waiting in Line...' screen
+	WaitInLineTimeout: 80, // Seconds to wait before cancelling the 'Waiting in Line...' screen
```

Comment:
I dont see a reason for this entire file to be edited for this specific pull request